src/components/repository-section-title.component.js

```diff
@@ -33,8 +35,11 @@ export const RepositorySectionTitle = ({
       <Text style={styles.titleText}>
         {text}
       </Text>
-      <StateBadge type="open" text={openCount} style={styles.badge} />
-      <StateBadge type="closed" text={closedCount} style={styles.badge} />
+      {!loading &&
+        <View style={styles.title}>
```

Comment:
`<View style={styles.title}>` looks duplicated here and in line 34. I'm assuming we need `flexDirection: 'row'` applied to both - if that's the case could we add a new style here for `styles.badgeContainer` or something along those lines and rename `styles.title` to `styles.container` where both can just be `flexDirection: 'row'`?